doc/developer/code-coverage.md

```diff
@@ -0,0 +1,50 @@
+# Code Coverage for Pull Requests
+
+## Motivation
+
+A full [code coverage report](http://65.109.125.29/coverage_01879007-43c4-49db-a81d-a44e93dfb55a/) allows you to check for each part of the code if there is a test that currently covers it by executing the relevant lines of code. Unfortunately the motivation to increase code coverage using full reports is low, and the effort to understand why no existing mzcompose-based test covers the code in question is high.
```

Comment:
I think you should avoid linking directly to this machine. I would have suggested that you link to a buildkite artifact, but those can expire , so I do not have a suitable link to suggest as a replacement at this time.

Reply:
I also thought about that and don't have a good place to put it yet, but wanted to have a demo report somewhere. Removed the link for now until we have a better solution.